### DISCUSSION TITLE
我是用CLI创建collection时，如何为什么不能使用STRING

### DISCUSSION THREAD

**Discussion by LiJie20190102** [milvus-io/milvus#23413] [Q&A and General discussion] [answered]
![image](https://user-images.githubusercontent.com/53458004/231689894-d64dcdcb-8505-442b-ba80-3007fa6ff112.png)

**Comment by nameczz** [comment, marked as answer]
Looks like it's wrong validation for field type in cli. Will fix it asap.

**Reply by LiJie20190102** [reply]
Okay, thank you
